packages/gradle/src/generators/init/init.ts

```diff
@@ -158,7 +158,7 @@ function updateNxJsonConfiguration(tree: Tree) {
   );
   const productionFileSet = nxJson.namedInputs.production ?? [];
   nxJson.namedInputs.production = Array.from(
-    new Set([...productionFileSet, 'default', '!{projectRoot}/test/**/*'])
+    new Set([...productionFileSet, 'default', '!{projectRoot}/src/test/**/*'])
```

Comment:
I think this needs a migration :thinking: 

Workspaces right now do not have the right production fileset.